### ISSUE TITLE
Using Brush Tool crashes Synfig consistently

### ISSUE THREAD

**Issue by FirasH** [synfig/synfig#1161]
**Synfig version & platform**:
Latest from master: 474fc9
Using openSUSE Leap 15.1

**Issue description**:
1. Start Synfig Studio
2. Select the "Brush Tool"
3. Click anywhere on canvas: a dialog will pop-up with title "Please choose an image file"
4. Click on "Cancel" or "Load"

Synfig crashes

**Comment by mohamedAdhamc** [comment]
yep, just noticed this also, was going to report it but found this issue.
